CONTRIBUTING.md

```diff
@@ -59,10 +59,41 @@ Issues are artifacts which are used to clarify and document specific problems. I
+- PRs should target the default branch for merge
+- PR titles should be a summary of what was implemented in the PR
+- The description of the PR should contain a reference to an issue which it closes using the [github notation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) e.g. `fixes #XX`, `fixes #XX, fixes #YY`
+- Context information for how the fix works and why things were breaking in the way that they were should be included in the PR description instead of commit messages
+- If the changes require updates to any documentation, diagrams or other content which needs to stay in sync or be added, this should be included in the PR as well
+- Applicable test cases or automation updates should be included in the PR
+- If any process or policy was not followed, notes should be provided to reviewers in the PR description explaining why
 
 ### Reviewing Pull Requests
 
-(( TODO: Explain how to review changes ))
-(( TODO: Explain what the requirements for approving are ))
+Our Pull Request review process values [optimistic merging](https://github.com/openpracticelibrary/openpracticelibrary/issues/208) from [Pieter Hintjens](http://hintjens.com/blog:106). While we don't adhere strictly to the [c4 process spec](https://rfc.zeromq.org/spec/42/#23-patch-requirements), the spirit of focusing on one problem at a time and making 'good enough' / 'correct' incremental progress is critical to our process.
+
+As reviewers, PRs should be approved and merged if they are `Correct` as defined by following criteria:
+
+1. All automated and manual validation tests pass on the topic branch on at least the primary platform
+1. The product builds successfully on at least the primary platform
+1. All applicable style and contribution guides are met
+1. The PR is a focused and accurate answer to one agreed on Issue
+
+In general, reviewers are acting in the role of maintainers in the open-source model and have the following responsibilities:
+
+1. Are respectful stewards of Correctness
+    1. Do not enforce rules outside of documentation
+    1. Teach Correctness rules by referencing documentation and stating how it applies (e.g. PR focus: these lines do not apply to fixing the Issue)
+    1. Update documentation and propose improvements based on reviewing experience
+    1. Update automation when it doesn't match documentation
+    1. Create automation to improve review process
+1. Use good judgement
+    1. Reduce their bias by seeking additional opinion
+    1. Don't enforce process to be pedantic, be flexible
+1. Are responsible for getting PRs merged
+    1. Make changes to fix correctness directly in branches
+    1. Follow up regularly on blockers to merging or open conversations
+    1. Create and encourage creation of issues to address new problems / fixes
+1. Share non-blocking improvement feedback
```

Comment:
Yeah, I was thinking about this a bit, too--

My initial thought is that since the reviewer is responsible for getting the PR merged, they know which conversations already that can be resolved and can resolve them when they think the PR is ready for merge.

Let's see how it goes in practice and fix it as we learn more.